### PR TITLE
Add note that seccomp 2.2.1 or higher is required

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @jfrazelle @moxiegirl for review

### CODE REVIEW

**Comment by calavera** [comment]
What about turning the sentence into positive, `this is **only** available with seccomp 2.2.2 and higher`?

File: docs/security/seccomp.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ feature to restrict your application's access.
 This feature is available only if the kernel is configured with `CONFIG_SECCOMP`
 enabled.
 
+> **Note**: this feature is not available on systems using seccomp 2.2.1
+> and older, such as Ubuntu 14.04, Debian Wheezy, and Debian Jessie
```
Comment (calavera) [inline docs/security/seccomp.md]: missing dot at the end of the sentence :smile_cat:

**Comment by thaJeztah** [comment]
@calavera yes, or only mention the distro's we support it on. Possibly, mention the option of downloading the static binary.

File: docs/security/seccomp.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ feature to restrict your application's access.
 This feature is available only if the kernel is configured with `CONFIG_SECCOMP`
 enabled.
 
+> **Note**: this feature is not available on systems using seccomp 2.2.1
```
Comment (moxiegirl) [inline docs/security/seccomp.md]: @thaJeztah spoke with Jess, this is what we came up with:

```markdown
>**Note**: On Ubuntu 14.04, Debian Wheezy, and Debian Jessie, You must download the [latest
static Docker Linux binary](../installation/binaries.md) to use seccomp.
```

**Comment by thaJeztah** [comment]
ping @moxiegirl updated, PTAL

**Comment by calavera** [comment]
LGTM

**Comment by thaJeztah** [comment]
author LGTM. Documentation failure is not related;

```
	Found: 1255 files
	Found: 0 errors
ERRORS = 0
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
running hugo
INFO: 2016/02/05 19:58:34 hugo.go:377: Using config file: config.toml
INFO: 2016/02/05 19:58:34 hugo.go:450: syncing from /docs/themes/hugo-foundation/static to /docs/public/
INFO: 2016/02/05 19:58:34 hugo.go:457: syncing from /docs/static/ to /docs/public/
initialize & template prep:
	34.099756ms (35.05221ms)	    6.47 MB 	16696 Allocs
load data:
	337.677µs (35.564192ms)	    0.02 MB 	187 Allocs
2016/02/05 19:58:35 mkdir /docs/public/engine/security/https/: no such file or directory
ERRORS = 1
```

**Pull request merged**